managed-databases/postgresql-and-mysql/how-to/create-a-database.mdx

```diff
@@ -40,13 +40,15 @@ Compared to traditional database management, which requires customers to provide
         </Message>
     - Select a node type.
     - Choose your storage type and capacity. Two types of storage are available:
+        - **Basic Block Storage** - With this type, your storage is decoupled from your compute resources. You can increase your storage space without changing your node type. You can define your storage capacity by entering the desired amount in the box.
+
+        - **Block Storage Low Latency**: This storage type works like the Basic Block Storage, while providing enhanced efficiency. You can choose between **5k** or **15k IOPS** according to your workloads input/output operations needs, with lower latency and high resiliency. You can increase your volume to up to 10 TB.
+
         - **Local SSD Storage** - Your storage is fixed and tied to your compute resource.
         <Message type="note">
           The local storage option is only available with the first generation node types.
         </Message>
-
-        - **Block SSD Storage** - With this type, your storage is decoupled from your compute resources. You can increase your storage space without changing your node type, with no downtime. You can define your storage capacity by entering the desired amount in the box. You can increase your volume to up to 10 TB.
-    - Enable automatic backups, if necessary.
+    - Select the snapshot configuration. You can either leave the default **Automatic** option to enable [autobackups](/managed-databases/postgresql-and-mysql/how-to/enable-autobackup), or select [Manual snapshots](/managed-databases/postgresql-and-mysql/how-to/manage-manual-backups).
```

Comment:
the last link point out on a link with logical backups doc whereas we want to explain how to perform manual snapshots which is not the same

Reply:
```suggestion
    - Select the snapshot configuration. You can either leave the default **Automatic** option to enable [autobackups](/managed-databases/postgresql-and-mysql/how-to/enable-autobackup), or select [Manual snapshots](/managed-databases/postgresql-and-mysql/how-to/manage-snapshots).
```